python/ray/job_config.py

```diff
@@ -55,16 +57,7 @@ def serialize(self):
         return self.get_proto_job_config().SerializeToString()
 
     def set_runtime_env(self, runtime_env: Optional[Dict[str, Any]]) -> None:
-        # Lazily import this to avoid circular dependencies.
-        import ray._private.runtime_env as runtime_support
-        if runtime_env:
-            self._parsed_runtime_env = runtime_support.RuntimeEnvDict(
-                runtime_env)
-            self.worker_env.update(
```

Comment:
did we skipped this part or the runtime env is already set ?

Reply:
skip which part?

Reply:
im referring to self.worker_env.update() that only gets called on the original one but removed in your change just to make sure we didn't drop this part by accident. By searching "env_vars" on this page i didn't find very related change either .. so how we're handing worker_env update in this function now ?

Reply:
Oh, gotcha. I accidentally deleted it because it's a deprecated option (actually I have another PR that removes it), but replaced it now. Good catch